skimage/transform/radon_transform.py

```diff
@@ -224,13 +247,20 @@ def iradon(
         Whether to keep the original range of values. Otherwise, the input
         image is converted according to the conventions of `img_as_float`.
         Also see https://scikit-image.org/docs/dev/user_guide/data_types.html
+    center : array_like, optional
+        It specifies the rotation center of the reconstructed image. This also
+        affects where it is supposed to be in the radon image. By default, the
+        rotation axis will be located in the reconstruction at the position:
+        ``(reconstructed.shape[0] // 2, reconstructed.shape[1] // 2)``
+        This will change in a future release, and it will be moved to:
+        ``((reconstructed.shape[0] - 1) / 2, (reconstructed.shape[1] - 1) / 2)``
+        Accordingly, it will be located at the position
```

Comment:
I think that it's not clear whether 'Accordingly' refers to the current default or the future default...